br/pkg/lightning/restore/meta_manager.go

```diff
@@ -78,6 +78,8 @@ func (b *dbMetaMgrBuilder) TableMetaMgr(tr *TableRestore) tableMetaMgr {
 type tableMetaMgr interface {
 	InitTableMeta(ctx context.Context) error
 	AllocTableRowIDs(ctx context.Context, rawRowIDMax int64) (*verify.KVChecksum, int64, error)
+	// re-allocate rowIDs across lightning instances
+	// only parallel import needs this
```

Comment:
Could add more comments to explain the return value? BTW, comments should begin with the method name.

Reply:
4df0f47